training/instructlab/Makefile

```
@@ -31,9 +31,9 @@ nvidia: instructlab
 .PHONY: amd
 amd: instructlab
 	rm -rf ../build/instructlab-$@
-	"${CONTAINER_TOOL}"  build --squash-all -t oci:../build/instructlab-$@ -f instructlab/containers/rocm/Containerfile instructlab
+	"${CONTAINER_TOOL}"  build --squash-all -t quay.io/ai-lab/instructlab-amd:latest -t oci:../build/instructlab-$@ -f instructlab/containers/rocm/Containerfile instructlab
 
 .PHONY: intel
 intel: instructlab
 	rm -rf ../build/instructlab-$@
-	"${CONTAINER_TOOL}"  build --squash-all -t oci:../build/instructlab-$@ -f instructlab/containers/hpu/Containerfile instructlab
+	"${CONTAINER_TOOL}"  build --squash-all -t oci:../build/instructlab-$@ -f instructlab/containers/hpu/Containerfile instructlab
```

Linting + add tagging for intel too:
```suggestion
	"${CONTAINER_TOOL}"  build --squash-all -t quay.io/ai-lab/instructlab-intel:latest -t -t oci:../build/instructlab-$@ -f instructlab/containers/hpu/Containerfile instructlab

```